packages/apps/human-app/frontend/src/modules/worker/components/sign-in/sign-in-form.tsx

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable camelcase -- ...*/
+import { FormProvider, useForm } from 'react-hook-form';
+import { Box, Grid, Typography } from '@mui/material';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { useTranslation } from 'react-i18next';
+import { Link } from 'react-router-dom';
+import { Input } from '@/shared/components/data-entry/input';
+import { Button } from '@/shared/components/ui/button';
+import { Password } from '@/shared/components/data-entry/password/password';
+import { routerPaths } from '@/router/router-paths';
+import { FormCaptcha } from '@/shared/components/h-captcha';
+import { type SignInDto } from '@/modules/worker/services/sign-in/types';
+import { signInDtoSchema } from '@/modules/worker/services/sign-in/schema';
+import { useResetMutationErrors } from '@/shared/hooks/use-reset-mutation-errors';
+
+interface SignInFormProps {
+  onSubmit: (data: SignInDto) => void;
+  error?: unknown;
+  isLoading?: boolean;
+  resetMutation: () => void;
+}
+
+export function SignInForm({
```

Comment:
Is this component reusable in `worker` module or is it only for `SignIn` page?
If it's only for sing in - then probably we shouldn't have it in `worker/components`